build.sbt

```diff
@@ -442,7 +442,9 @@ lazy val codegenPlugin = (projectMatrix in file("modules/codegen-plugin"))
         // for the code being built
         (`aws-kernel`.jvm(Scala213) / publishLocal).value,
         (core.jvm(Scala213) / publishLocal).value,
+        (core.jvm(Scala3) / publishLocal).value,
         (dynamic.jvm(Scala213) / publishLocal).value,
+        (dynamic.jvm(Scala3) / publishLocal).value,
```

Comment:
is that a mistake or you do need dynamic to be published locally for sbt-codegen to be testable?

Reply:
sorry, yes this is a mistake